docs/how-to/data-layer.txt

```diff
@@ -13,69 +13,69 @@ solutions for performing many common deep learning workflows including:
 * :ref:`Distributed Training<multi-gpu-training>`
 
 
-Determined integrates the `data-layer` with `YogaDL <https://github.com/determined-ai/yogadl>`_ to provide a better
-approach to data-loading. The data-layer enables efficient data access for common deep learning
+Determined integrates the `data layer` with `YogaDL <https://github.com/determined-ai/yogadl>`_ to provide a better
+approach to data-loading. The data layer enables efficient data access for common deep learning
 workflows such as hyperparameter tuning and distributed training. It also enables dataset
 versioning. This how-to guide covers:
 
-#. How the data-layer works.
+#. How the data layer works.
 #. How to configure a local file-system, `AWS S3 <https://aws.amazon.com/s3/>`_, or `GCS <https://cloud.google.com/storage>`_
    as the storage medium.
-#. How to use the data-layer API in your code.
+#. How to use the data layer API in your code.
 
 
 .. note::
-    The data-layer is an experimental feature and its API is not considered stable. Currently it supports only
+    The data layer is an experimental feature and its API is not considered stable. Currently it supports only
     ``tf.data.Dataset`` inputs, which can be used in :ref:`Keras <tf-keras-trial>`
     and :ref:`Estimators <estimator-trial>`.
 
 
-How the Data-Layer Works
+How the Data layer Works
```

Comment:
blocking: This capitalization feels odd? "How the Data layer Works" suggest going title-case or sentence-case, not both?

Reply:
Whoops, that was a bad find-and-replace. You're totally right.